services/firebase/config.js

```diff
@@ -5,8 +5,9 @@
 // For Firebase JS SDK v7.20.0 and later, measurementId is optional
 
 import { initializeApp } from 'firebase/app';
-import { initializeAuth, getReactNativePersistence } from 'firebase/auth';
-import ReactNativeAsyncStorage from '@react-native-async-storage/async-storage';
+// import { initializeAuth, getReactNativePersistence } from 'firebase/auth';
+import { ReactNativeAsyncStorage } from '@react-native-async-storage/async-storage';
+import { initializeAuth, getReactNativePersistence } from '@firebase/auth';
```

Comment:
@Miguel It doesn't resolve the issue: When I run the app on the web, `getReactNativePersistence` is still not recognized